vhost-device-i2c/README.md

```diff
@@ -33,15 +33,27 @@ Examples section below.
 
 .. option:: -l, --device-list=I2C-DEVICES
 
-  I2c device list at the host OS in the format:
-      <bus-name>:<client_addr>[:<client_addr>],[<bus-name>:<client_addr>[:<client_addr>]]
+  I2c device list at the host OS can be in two different format, name and number:
 
+- format by name:
+      \<bus-name>:<client_addr>[:<client_addr>],[\<bus-name>:<client_addr>[:<client_addr>]]
+```
       Example: --device-list "i915 gmbus dpd:32:21,DPDDC-D:10:23"
-
+```
   Here,
       bus-name: is adatper's name. e.g. value of /sys/bus/i2c/devices/i2c-0/name.
       client_addr (decimal): address for client device, 32 == 0x20.
 
+- format by number:
+    \<bus>:<client_addr>[:<client_addr>],[\<bus>:<client_addr>[:<client_addr>]]
```

Comment:
The ‘address’ is typically used to refer to the address of an I2C slave device. It may not be the best choice.
In the Linux, the ‘0’ in /dev/i2c-0 might be described as an [I2C Bus Physical Number](https://docs.kernel.org/i2c/i2c-sysfs.html#i2c-bus-physical-number). 
What's more, 'adapter_no' has been used before, so I prefer using bus number rather than bus-id or bus-addr.

Reply:
okay, makes sense. Sorry, I'm not an i2c expert ;-)